spec/models/achievement_spec.rb

```diff
@@ -39,6 +39,17 @@
     end
   end
 
+  describe "to_s" do
+    it "is well-formed" do
+      subject.side = 'corp'
+      subject.title = 'Membrane'
+      subject.points = 2
+      subject.description = 'Whatever'
+
+      expect(subject.to_s).to eq "Achievement [corp/2] 'Membrane' / 'Whatever'"
+    end
+  end
```

Comment:
Feels a bit overkill to add a test for a `to_s` method but I didn't want the pull request to be rejected because I brought the code coverage % down.

:smile_cat: